### DISCUSSION TITLE
Why is the performance gap between `cockroachdb` and `yugabytedb` so large？

### DISCUSSION THREAD

**Discussion by microyahoo** [cockroachdb/cockroach#83769] [Q&A] [answered]
The `yugabyteDB` official document do the benchmark between cockroach and yugabyte, which shows yugabyteDB delivers 3x higher throughput on average, and yugabyteDB delivers 4.5x lower latency on average. Why is the performance gap between `cockroachdb` and `yugabytedb` so large?

[YUGABYTEDB VS COCKROACHDB](https://www.yugabyte.com/yugabytedb-vs-cockroachdb/)

**Comment by jordanlewis** [comment, marked as answer]
Hi @microyahoo, make sure to validate any benchmarking claims (especially ones that seem outlandish or unlikely) you see on the internet by performing tests with your own workload. You shouldn't blindly trust companies that post benchmarking claims about their competitors, because of the obvious conflicts of interest therein.

Also, feel free to read this blog post (from 2019).

https://www.cockroachlabs.com/blog/unpacking-competitive-benchmarks/
